pkg/ebpf/tracer.go

```diff
@@ -369,7 +369,19 @@ func (t *Tracer) GetActiveConnections(clientID string) (*network.Connections, er
 	conns := t.state.Connections(clientID, latestTime, latestConns, t.reverseDNS.GetDNSStats())
 	names := t.reverseDNS.Resolve(conns)
 
-	return &network.Connections{Conns: conns, DNS: names}, nil
+	kprobeStats := GetProbeTotals()
```

Comment:
Can we split the creating of ConnectionsTelemetry into a separate function (and add error handling for bad casts, missing map keys)